daemon/src/lib.rs

```diff
@@ -47,6 +47,7 @@ pub mod projection;
 pub mod rollover_taker;
 pub mod routes;
 pub mod seed;
+pub mod send_future_ext;
```

Comment:
```suggestion
pub mod xtra_ext;
```

Probably go after the crate.